.github/workflows/build-and-test.yml

```diff
@@ -137,10 +137,18 @@ jobs:
   unittest-matrix:
     strategy:
       matrix:
+        runner: [ubuntu-latest, actuated-arm64-4cpu-4gb]
         go-version: ["~1.22", "~1.21.8"] # 1.20 needs quotes otherwise it's interpreted as 1.2
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.runner }}
     needs: [setup-environment]
     steps:
+      - name: Set up arkade
+        uses: alexellis/setup-arkade@master
```

Comment:
We should pin this to a tag or, in the absence of tags, to a specific commit

Reply:
That's fair ; updated.